dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/infer/InferrerHelper.java

```diff
@@ -125,6 +125,13 @@ protected boolean isDescriptorPath(String path) {
                     return path.endsWith(".ga");
                 }
             };
+        case SMK:
+            return new DescriptorLanguageInferrer(DescriptorLanguage.SMK) {
+                @Override
+                protected boolean isDescriptorPath(String path) {
+                    return path.endsWith(".snakemake-workflow-catalog.yml");
```

Comment:
`isDescriptorPath` currently identifies paths that point at a descriptor file, for the purpose of creating a pool of descriptors from which we determine one or more primary descriptors.  For languages like CWL and WDL, that's files with extensions `.cwl` or `.wdl`.  For Snakemake, IMHO, the `Snakefile` is always the single primary descriptor, and assuming that's true, we should check for it here.

Reply:
See conversation at https://github.com/dockstore/snakemake-language-interface/pull/2#discussion_r2017476964

Reply:
If the primary descriptor for a Snakemake workflow is indeed the `Snakefile`, would recommend changing this to match, otherwise the inference is going to fail, because it will identify the catalog file as the primary descriptor.